.github/workflows/deployment.yml

```diff
@@ -286,3 +289,34 @@ jobs:
             az functionapp deployment slot delete \
             --slot ${{ github.sha }} \
             --name ${{ env.FUNCTION }}
+
+  sql-migrations:
```

Comment:
Discussed in person:
- move migrations to before code deployment
- on deployment script look at making it an approval needed skippable action in the pipeline

Reply:
As discussed, we'll hold off adding a step specific approval step for now.